bitwarden_access_manager.py

```diff
@@ -0,0 +1,737 @@
+        )
+        bearer_token = response.json()["access_token"]
+        return BitwardenClient(HTTPSConnection("api.bitwarden.com"), bearer_token)
+
+    def _http_get(self, url: str) -> HTTPResponse:
+        self.connection.request(
+            method="GET",
+            url=url,
+            headers={
+                "Accept": "application/json",
+                "Authorization": f"Bearer {self.bearer_token}",
+            },
+        )
+
+        return self.connection.getresponse()
+
+    def get_groups(self) -> Iterable[Group]:
+        groups = json.load(self._http_get(f"/public/groups"))
+        for group in groups["data"]:
+            yield Group(
+                id=group["id"],
+                name=group["name"],
+                access_all=group["accessAll"],
+            )
+
+    def get_collection_groups(self, groups: Any) -> Iterable[GroupCollectionAccess]:
+        for group in groups:
+            access = self.check_access(group["readOnly"])
+
+            group_id = group["id"]
+            yield GroupCollectionAccess(
+                name=json.load(self._http_get(f"/public/groups/{group_id}"))["name"],
+                access=access,
+            )
+
+    def get_collections(
+        self,
+        org_members: Dict[str, Member],
+        collections_members: Dict[str, List[MemberCollectionAccess]],
+    ) -> Iterable[Collection]:
+        collections = json.load(self._http_get(f"/public/collections"))
+
+        for collection in collections["data"]:
+            group_accesses: Optional[Tuple[GroupCollectionAccess, ...]] = None
+            member_accesses: Optional[Tuple[MemberCollectionAccess, ...]] = None
+            collection_id = collection["id"]
+
+            collection_data = json.load(
+                self._http_get(f"/public/collections/{collection_id}")
+            )
+
+            group_accesses_data = tuple(
+                sorted(self.get_collection_groups(collection_data["groups"]))
+            )
+
+            if len(group_accesses_data) > 0:
+                group_accesses = group_accesses_data
+
+            if collection_id in collections_members:
+                member_accesses = tuple(sorted(collections_members[collection_id]))
+
+            yield Collection(
+                id=collection["id"],
+                external_id=collection["externalId"],
+                member_access=member_accesses,
+                group_access=group_accesses,
+            )
+
+    def get_group_members(self, id: str, name: str) -> Iterable[GroupMember]:
+        members = json.load(self._http_get(f"/public/groups/{id}/member-ids"))
+
+        for member in members:
+            member = json.load(self._http_get(f"/public/members/{member}"))
+            yield GroupMember(
+                member_id=member["id"],
+                member_name=member["name"],
+                group_name=name,
+            )
+
+    def set_member_type(self, type_id: int) -> MemberType:
```

Comment:
```suggestion
    def map_member_type(self, type_id: int) -> MemberType:
```
This method does not set anything.